Major upgrade 1 (#797)
o1-labs/docs2: docs/node-operators/index.mdx

Node operators are network participants who run mina nodes on a Mina network.

In this section we describe operations on Mina and how the Node Operators can run Mina nodes.

- [Staking and Snarking](/node-operators/staking-and-snarking) - How to stake your Mina
- [Generating a Key Pair](/node-operators/generating-a-keypair) - How to generate key pairs
- [Block Producing Node](/node-operators/block-producer-node) - Running a Block Producer
- [SNARK Workers](/node-operators/snark-workers) - Running a SNARK worker
- [Archive Nodes](/node-operators/archive-nodes) - Running an Archive Node
- [Seed Peers](/node-operators/seed-peers) - Running a Seed Peer
- [Data and History](/node-operators/data-and-history) - Lets look at retreiving historic data.
- [Delegation Program](/node-operators/delegation-program) - The Mina Foundation Delegation Prgram for Block Producers 
- [Staking Service Guidelines](/node-operators/staking-service-guidelines)
- [Mina Signer](/node-operators/mina-signer)
- [Mina CLI Reference](/node-operators/mina-cli-reference) - Guide to CLI interactions with Mina networks 
- [Troubleshooting](/node-operators/troubleshooting)
- [FAQ](/node-operators/faq)

## Mina Nodes

> Mina nodes fulfill different roles within the network:

Reviewer: ```suggestion
A node is a machine running the Mina daemon. Different nodes fulfill different roles within the Mina network:
```

> 1. Block Producer - this is analogous to being a Bitcoin "miner" or a "validator" in other proof-of-stake networks. By staking MINA, you can be selected to produce a block and win the block reward.

Reviewer: oh, hmmm, this section is about node types. But here it seems as if we are talking about the person block producer. 
See 
https://docs.minaprotocol.com/glossary#block-producer

> 1. Block Producer - this is analogous to being a Bitcoin "miner" or a "validator" in other proof-of-stake networks. By staking MINA, you can be selected to produce a block and win the block reward.

Reviewer: ```suggestion
1. Block Producer - A node that participates in a process to determine what blocks it is allowed to produce and then produces blocks containing transactions that can be broadcast to the network. People who run [block producer](https://docs.minaprotocol.com/mina-protocol/block-producers) nodes are also called block producers.
```

> 2. SNARK Coordinators - distribute work to a series of SNARK workers in parallel to block production.

Reviewer: ```suggestion
2. [SNARK Coordinators](https://docs.minaprotocol.com/glossary#snark-coordinator) - A role on a Mina node in the Mina network that distributes work to a series of SNARK workers in parallel to block production.
```

Reviewer: wdyt linking to the glossary? and let's fix up the glossary if updates are required

3. Snark Workers -  Snark workers create [zk-snarks](https://minaprotocol.com/blog/what-are-zk-snarks) for each transaction. These `zk-snarks` are used to create recursive `zk-snarks` proving the correctness of a block and in turn these `zk-snarks` are used to create a recursive `zk-snark` which proves the the correctness of the network. These `zk-snark` help provide the Mina Protocol with succinctness. 

4. Archive Nodes - A Mina node configued to store historical chain data to a persistent data source, currently PostgreSQL

> 5. Seed Nodes - Keep a record of nodes in the network and enable nodes joining the network to connect to peer nodes

Reviewer: ```suggestion
5. Seed Nodes - Keep a record of nodes in the network and enable nodes joining the network to connect to peer nodes.
```

## Mina Protocol

> [Mina Protocol](../mina-protocol) describes how the MIna Protocol works.
> 
> ## Node Developers
> 
> [Node Developers](../node-developers) are developers who add to and improve mina nodes.

Reviewer: ```suggestion
[Mina Protocol](../mina-protocol) describes how the Mina Protocol works.

## Node Developers

[Node Developers](../node-developers) are developers who add to and improve Mina nodes.
```

## Exchange Operators

> [Exchange Operators](../exchange-operators/faq) connect to mina networks to provide access to the mina token.

Reviewer: ```suggestion
[Exchange Operators](../exchange-operators/faq) connect to Mina networks to provide access to the MINA token.
```
